persistence-modules/spring-boot-h2/spring-boot-h2-database/src/main/java/com/baeldung/naming/MetadataExtractorIntegrator.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.naming;
+
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.model.relational.Database;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
+import org.hibernate.service.spi.SessionFactoryServiceRegistry;
+
+public class MetadataExtractorIntegrator implements org.hibernate.integrator.spi.Integrator {
```

Comment:
Thanks for your explanation in the other PR that this is for convenience in operating the demonstration. Let's place a comment block at the class level here explaining that. The reason is so that folks don't inadvertently use it, thinking it is necessary.

Reply:
Good, I'll add the comment.